packages/authentication/modules/access/useSignUp/index.ts

```diff
@@ -1,13 +1,13 @@
 'use client'
```

Comment:
You shoudln't be here, this file was removed. SignUp is now handled in useAllAuthSignUp. Did you base this off the correct branch? `BA-1786-fe-mfa-setup-fe-library`